### PR TITLE
New initialization that will be used for refactored tests

### PR DESCRIPTION
**Pull request by jacekchalupka**

### CODE REVIEW

File: CIScripts/Testenv/Testenv.Impl.ps1 [hunk 1]
```diff
@@ -4,18 +4,64 @@ class Testenv {
...
+
+    [Void] Initialize() {
+        $CleanupStack = $this.NewCleanupStack()
+        $this.Sessions = New-RemoteSessions -VMs $this.Testbeds
+        $CleanupStack.Push({Param([PSSessionT[]] $Sessions) Remove-PSSession $Sessions}, @(, $this.Sessions))
+        Set-ConfAndLogDir -Sessions $this.Sessions
+        Sync-MicrosoftDockerImagesOnTestbeds -Sessions $this.Sessions
+    }
+
+    [Void] Initialize_New([String] $LogDir, [String] $ContrailProject, [Bool] $InstallComponents) {
```
Comment (jacekchalupka) [inline CIScripts/Testenv/Testenv.Impl.ps1]: This method will be joined with Initialize after all tests will be refactored

Comment (aethyrxeo) [reply CIScripts/Testenv/Testenv.Impl.ps1]: Add TODO comment please

**Comment by jacekchalupka** [comment]
recheck no bug

**Comment by jacekchalupka** [comment]
recheck no bug

**Pull request merged**